cups.py

```diff
@@ -207,6 +206,24 @@ def info( args ):
 
     
 
+def update_production_status( update_query, retries=10, delay=10.0 ):
```

Comment:
This looks like a zombie method... removed in previous PRs, but this PR is bringing
it back from the dead.